manifest.json

```diff
@@ -32,7 +32,10 @@
     },
     "projectOptions": ["app", "addon"],
     "nodeVersion": "4.0.0",
-    "commands": ["github:rondale-sc/es5-getter-ember-codemod es5-getter-ember-codemod app/**/*.js"]
+    "commands": [
+      " github:rondale-sc/es5-getter-ember-codemod es5-getter-ember-codemod app/**/*.js",
```

Comment:
this is in the es5-getter-ember-codemod block, you should make a new block.

Reply:
New block added for ember-3x-codemods